<ui version="4.0" >
<comment> Begin CVS Header </comment>
<comment>   $Source: /Volumes/Home/Users/shoops/cvs/copasi_dev/copasi/UI/CQEventWidget1.ui,v $ </comment>
<comment>   $Revision: 1.13 $ </comment>
<comment>   $Name:  $ </comment>
<comment>   $Author: shoops $ </comment>
<comment>   $Date: 2011/03/29 16:17:20 $ </comment>
<comment> End CVS Header </comment>

<comment> Copyright (C) 2011 - 2010 by Pedro Mendes, Virginia Tech Intellectual </comment>
<comment> Properties, Inc., University of Heidelberg, and The University </comment>
<comment> of Manchester. </comment>
<comment> All rights reserved. </comment>

<comment> Copyright (C) 2008 by Pedro Mendes, Virginia Tech Intellectual </comment>
<comment> Properties, Inc., EML Research, gGmbH, University of Heidelberg, </comment>
<comment> and The University of Manchester. </comment>
 <class>CQEventWidget1</class>
 <widget class="CopasiWidget" name="CQEventWidget1" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>504</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="caption" stdset="0" >
   <string>Event Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QGridLayout" name="gridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="mpLabelEventName" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Name</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3" >
      <widget class="QLineEdit" name="mpLineEditName" />
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="mpLabelTrigger" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Trigger Expression</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
       </property>
       <property name="wordWrap" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="3" >
      <widget class="CQExpressionMmlStackedWidget" name="mpExpressionTrigger" >
       <property name="minimumSize" >
        <size>
         <width>380</width>
         <height>100</height>
        </size>
       </property>
       <property name="currentIndex" >
        <number>1</number>
       </property>
       <widget class="QWidget" name="page_11" />
       <widget class="QWidget" name="page_12" />
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QCheckBox" name="mpCheckDelay" >
       <property name="layoutDirection" >
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text" >
        <string>Delay</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QRadioButton" name="mpBtnDelayCalculation" >
       <property name="text" >
        <string>Calculation</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2" >
      <widget class="QRadioButton" name="mpBtnDelayAssignment" >
       <property name="text" >
        <string>Assignment</string>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="3" >
      <spacer name="mpSpacerDelay" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0" >
      <widget class="QLabel" name="mpLabelDelayExpression" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Expression</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="3" >
      <widget class="CQExpressionMmlStackedWidget" name="mpExpressionDelay" >
       <property name="minimumSize" >
        <size>
         <width>380</width>
         <height>100</height>
        </size>
       </property>
       <widget class="QWidget" name="page_7" />
       <widget class="QWidget" name="page_8" />
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="mpLine1" >
     <property name="frameShape" >
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" >
     <item row="1" column="0" >
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QListWidget" name="mpLBTarget" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" >
         <item>
          <widget class="QToolButton" name="mpBtnSelectObject" >
           <property name="maximumSize" >
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="toolTip" >
            <string>edit target</string>
           </property>
           <property name="text" >
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="mpBtnAddTarget" >
           <property name="maximumSize" >
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="toolTip" >
            <string>add target</string>
           </property>
           <property name="text" >
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="mpBtnDeleteTarget" >
           <property name="maximumSize" >
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="toolTip" >
            <string>delete target</string>
           </property>
           <property name="text" >
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacerTargetIcons" >
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="0" column="0" >
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QLabel" name="mpLabelTarget" >
         <property name="text" >
          <string>Target</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacerTarget" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="0" column="1" >
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QLabel" name="mpLabelEA" >
         <property name="text" >
          <string>Expression</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacerExpression" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="1" column="1" >
      <widget class="CQExpressionMmlStackedWidget" name="mpExpressionEA" >
       <property name="minimumSize" >
        <size>
         <width>300</width>
         <height>60</height>
        </size>
       </property>
       <widget class="QWidget" name="page" />
       <widget class="QWidget" name="page_2" />
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="mpLine2" >
     <property name="frameShape" >
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="mpBtnCommit" >
       <property name="text" >
        <string>Commit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mpBtnRevert" >
       <property name="text" >
        <string>Revert</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mpBtnNew" >
       <property name="text" >
        <string>New</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mpBtnDelete" >
       <property name="text" >
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>CopasiWidget</class>
   <extends>QWidget</extends>
   <header>copasi/UI/copasiWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CQExpressionMmlStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header>CQExpressionMmlStackedWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mpLineEditName</tabstop>
  <tabstop>mpCheckDelay</tabstop>
  <tabstop>mpBtnDelayCalculation</tabstop>
  <tabstop>mpBtnDelayAssignment</tabstop>
  <tabstop>mpLBTarget</tabstop>
  <tabstop>mpBtnSelectObject</tabstop>
  <tabstop>mpBtnAddTarget</tabstop>
  <tabstop>mpBtnDeleteTarget</tabstop>
  <tabstop>mpBtnCommit</tabstop>
  <tabstop>mpBtnRevert</tabstop>
  <tabstop>mpBtnNew</tabstop>
  <tabstop>mpBtnDelete</tabstop>
 </tabstops>
 <includes>
  <include location="global" >string</include>
  <include location="global" >vector</include>
  <include location="local" >copasi/UI/copasiWidget.h</include>
  <include location="local" >MyLineEdit.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>mpBtnCommit</sender>
   <signal>clicked()</signal>
   <receiver>CQEventWidget1</receiver>
   <slot>slotBtnCommitClicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpBtnDelete</sender>
   <signal>clicked()</signal>
   <receiver>CQEventWidget1</receiver>
   <slot>slotBtnDeleteClicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpBtnNew</sender>
   <signal>clicked()</signal>
   <receiver>CQEventWidget1</receiver>
   <slot>slotBtnNewClicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpBtnRevert</sender>
   <signal>clicked()</signal>
   <receiver>CQEventWidget1</receiver>
   <slot>slotBtnRevertClicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpBtnAddTarget</sender>
   <signal>clicked()</signal>
   <receiver>CQEventWidget1</receiver>
   <slot>slotAddTarget()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpBtnDeleteTarget</sender>
   <signal>clicked()</signal>
   <receiver>CQEventWidget1</receiver>
   <slot>slotDeleteTarget()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpBtnSelectObject</sender>
   <signal>clicked()</signal>
   <receiver>CQEventWidget1</receiver>
   <slot>slotSelectObject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
